src/index.html

```diff
@@ -2,16 +2,196 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset>
+        <legend>Personal information</legend>
+        <div class="form-group">
+          <label for="surname">Surname:</label>
+          <input
+              type="text"
+              name="surname"
+              id="surname"
+              autocomplete="off"
+              required
+          >
+        </div>
+
+        <div class="form-group">
+          <label for="name">Name:</label>
+          <input
+              type="text"
+              name="name"
+              id="name"
+              autocomplete="off"
+              required
+          >
+        </div>
+
+        <div class="form-group">
+          <label for="age">How old are You?</label>
+          <input
+              type="number"
+              name="age"
+              id="age"
+              max="100"
+              min="1"
+              value="12"
+          >
+        </div>
+
+        <div class="form-group">
+          <label for="dob">Full date of birth:</label>
+          <input
+              type="date"
+              name="dob"
+              id="dob"
+              required
+          >
+        </div>
+
+        <div class="form-group">
+          <label for="terms">I accept the term of the agreement</label>
+          <input
+             type="checkbox"
+             name="terms"
+             id="terms"
+             required
+          >
+        </div>
+
+      </fieldset>
```

Comment:
```suggestion
        </div>
      </fieldset>
```
Don't leave newlines between parent and child elements